projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/KubernetesService.java

```diff
@@ -2202,6 +2202,9 @@ public static V1Container container(
                     .withRequests(resources(request))
                     .withLimits(resources(limit))
                     .build())
+            .withEnvFrom(
+                new V1EnvFromSource()
+                    .secretRef(new V1SecretEnvSource().name("builder-secrets").optional(true)))
```

Comment:
losing track of what PR depends on what PR, and how are those linked to a ticket; I hope you are linking them somehow